tests/requirements/base.txt

```diff
@@ -6,6 +6,8 @@ jinja2 >= 2.7
 numpy
 Pillow
 PyYAML
+# Remove if it ends up being possible to build libmemcached on the Windows Jenkins box.
```

Comment:
I don't think so. The error is the same as what's here: http://stackoverflow.com/questions/32425121/cannot-install-pylibmc-for-python-on-windows-7-64-bit